public/locales/en/translation.json

```diff
@@ -99,7 +99,6 @@
                 "title-finished": "Update Finished"
             }
         },
-        "launchAnyway": "Launch Game Anyway",
```

Comment:
I think this file includes changes unrelated to this PR

I agree we should probably remove the pre-shader warning dialog now that umu doesn't require it, but it should be a different PR